doc/classes/AudioEffectEQCustom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectEQCustom" inherits="AudioEffectEQ" category="Core" version="3.2">
+	<brief_description>
+		A custom equalizer that allows band frequencies to be customized and changed on the fly.
+		Use it to create a custom equalizer if [AudioEffectEQ6], [AudioEffectEQ10] or [AudioEffectEQ21] don't fit your needs.
+	</brief_description>
+	<description>
+		[AudioEffectEQCustom] is a semi-parametric equalizer that allows a custom setup of band frequencies. The setup can be changed in real time as well.
+		An equalizer should contain at least 2 bands and at most 31 bands. The center frequencies should lie between 10 Hz and 22050 Hz (inclusive).
+		It is suggested to ensure that band frequencies strictly increasing.
```

Comment:
Something's missing, I think. "It is suggested to ensure that band frequencies **are** strictly increasing."